app/LoginPage/page.js

```diff
@@ -0,0 +1,82 @@
+"use client";
+import React, { useState } from "react";
+import { FiShoppingCart } from "react-icons/fi";
+import { FcGoogle } from "react-icons/fc";
+import { createUserWithEmailAndPassword, signInWithPopup } from "firebase/auth";
+import { auth, provider } from "../../FirebaseConfig/Firebase";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
```

Comment:
[Question]
Why do you have to save the email/password?

You should never save users' passwords locally.

Reply:
I did not store the login credentials locally, i used the state to pass the email and password to firebase